src/DataProtection/docs/Set-AzDataProtectionMSIPermission.md

```diff
@@ -0,0 +1,192 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/set-azdataprotectionmsipermission
+schema: 2.0.0
+---
+
+# Set-AzDataProtectionMSIPermission
+
+## SYNOPSIS
+Initializes Backup instance Request object for configuring backup
+
+## SYNTAX
+
+```
+Set-AzDataProtectionMSIPermission -BackupInstance <IBackupInstanceResource> -PermissionsScope <String>
+ -VaultName <String> -VaultResourceGroup <String> [-BypassPrompts <String>] [-KeyvaultId <String>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Initializes Backup instance Request object for configuring backup
+
+## EXAMPLES
+
+### Example 1: Grant Permissions for Azure Disks
+```powershell
+PS C:\> Set-AzDataProtectionMSIPermission -BackupInstance $instance -VaultResourceGroup "VaultRG"" -VaultName "Vaultname"" -PermissionsScope "ResourceGroup"
+
+```
+
+The above command is used to assign permissions to the backup vault "Vaultname" under resource group "VaultRG" at the "Rresource Group" scope of the disk.
+
+### Example 2: Grant Permissions for Azure Blobs
+```powershell
+PS C:\> Set-AzDataProtectionMSIPermission -BackupInstance $instance -VaultResourceGroup "VaultRG"" -VaultName "Vaultname"" -PermissionsScope "ResourceGroup"
```

Comment:
you can make this example slightly different than example 1, for example at a different scope